### PR TITLE
style(WelcomeCommand): Changed the wording to reflect current status.

### PR DESCRIPTION
**Pull request by Samantha-uk**
**What's the problem this PR addresses?**
Brings the wording of the welcome command to more accurately reflect the current status of Yarn 2.

...

**How did you fix it?**
Changed wording.

...

**Checklist**

- [x] I have read the [Contributing Guide](https://yarnpkg.com/advanced/contributing).

- [x] I have set the packages that need to be released for my changes to be effective.

- [x] I will check that all automated PR checks pass before the PR gets reviewed.

### CODE REVIEW

File: packages/yarnpkg-cli/sources/tools/WelcomeCommand.ts [hunk 1]
```diff
@@ -20,14 +21,15 @@ use the following URLs to get some insight:
   - Our Discord server:
     ${formatUtils.pretty(configuration, `https://discord.gg/yarnpkg`, `cyan`)}
 
-We're hoping you will enjoy the experience. For now, a good start is to run
-the two following commands:
+We hope you will enjoy your experience with Yarn 2.
+
+A good start is to run the two following commands:
 
   ${formatUtils.pretty(configuration, `find . -name node_modules -prune -exec rm -r {} \\;`, `magenta`)}
```
Comment (merceyz) [inline packages/yarnpkg-cli/sources/tools/WelcomeCommand.ts:28]: We should remove this line as the `node_modules` linker is a thing now and the PnP linker will clean them up anyways

Comment (Samantha-uk) [reply packages/yarnpkg-cli/sources/tools/WelcomeCommand.ts:28]: Do you think that replacing it by `yarn plugin list` and /or `yarn info` would be of value?  Or even the simple yarn -h?

**Comment by arcanis** [comment]
That's an old message! For context, this command originated from a feedback from @orta who mentioned it would be nice to print a message explaining the following steps when migrating from Yarn 1 -> 2 (we didn't have the migration guide yet). I thought this was a great idea.

What do you think about removing it and replacing it by a message when running `yarn install` with a legacy lockfile inside a TTY? Something like this:

```
❯ berry git:(master) ✗ ❯ yarn

➤ YN0000: Welcome on Yarn 2! Since it seems like you're migrating a project, do you want to open the migration guide? [Y/n]

➤ YN0000: ┌ Resolution step
➤ YN0002: │ @theme-ui/theme-provider@npm:0.4.0-highlight.0 doesn't provide @theme-ui/css (p9f0a8), requested by @theme-ui/mdx
➤ YN0002: │ @theme-ui/theme-provider@npm:0.4.0-highlight.0 doesn't provide react (pf58ce), requested by @emotion/core
➤ YN0002: │ @theme-ui/theme-provider@npm:0.4.0-highlight.0 doesn't provide react (p06fb8), requested by @theme-ui/core
➤ YN0002: │ @theme-ui/theme-provider@npm:0.4.0-highlight.0 doesn't provide react (p7b5d8), requested by @theme-ui/color-modes
➤ YN0002: │ @theme-ui/theme-provider@npm:0.4.0-highlight.0 doesn't provide react (p7cf72), requested by @theme-ui/mdx
➤ YN0002: │ @types/reflexbox@npm:4.0.1 doesn't provide @emotion/core (p71832), requested by @emotion/styled
...
```

If the user press enter or `Y`, we open https://yarnpkg.com/getting-started/migration, otherwise we just continue as usual.

**Comment by merceyz** [comment]
I don't think we should have any prompts that requires interaction on install, that's how you end up with https://classic.yarnpkg.com/en/docs/cli/install#toc-yarn-install-non-interactive

**Comment by arcanis** [comment]
Fair enough - just a link then?

**Comment by merceyz** [comment]
If we put it at the end of the install it should be visible regardless

**Comment by arcanis** [comment]
It's more difficult to print at the end since we need to deal with exception during the execution, etc.

**Comment by Samantha-uk** [comment]
Also ... is there anything in the migration guide that users should account for before running a Yarn berry install?

**Comment by arcanis** [comment]
Sorry I missed your message @Samantha-uk! I removed the `welcome` command in #2657 (it's almost never seen in its current state) so I'll close this PR, but feel free to ping us on Discord if you want to chat about other fun areas of improvements! 😃

**Pull request closed without merging**